Git insteadof config being respected results in missing repository.

### What happened?

If `git config --global url.git@github.com:.insteadof https://github.com/` is set, all requests to `https://github.com/user/repo.git` will be redirected to `git@github.com:user/repo.git`. This will cause github to request ssh credentials that cannot be supplied by the git library.

### Screenshots

![image](https://github.com/user-attachments/assets/e1a53aaa-cc10-42e3-b892-93ed21a2329b)



### Log output

<details>

```
> Task :net.fabricmc.devlaunchinjector.Main.main()
[14:41:54] [main/WARN] (FabricLoader/Knot) Class path entries reference missing files: /home/nea/src/Firmament/build/generated/ksp/main/classes, /home/nea/src/Firmament/build/classes/java/yacl, /home/nea/src/Firmament/build/generated/ksp/yacl/classes, /home/nea/src/Firmament/build/classes/kotlin/wildfireGender, /home/nea/src/Firmament/build/generated/ksp/wildfireGender/classes, /home/nea/src/Firmament/build/resources/wildfireGender, /home/nea/src/Firmament/build/classes/java/modmenu, /home/nea/src/Firmament/build/generated/ksp/modmenu/classes, /home/nea/src/Firmament/build/resources/modmenu, /home/nea/src/Firmament/build/generated/ksp/rei/classes, /home/nea/src/Firmament/build/classes/java/moulconfig, /home/nea/src/Firmament/build/generated/ksp/moulconfig/classes, /home/nea/src/Firmament/build/generated/ksp/texturePacks/classes - the game may not load properly!
[14:41:54] [main/INFO] (FabricLoader/GameProvider) Loading Minecraft 1.21.4 with Fabric Loader 0.16.9
[14:41:54] [ForkJoinPool-1-worker-9/WARN] (FabricLoader/Metadata) Mod fake-jarvis-api uses the version jarvis which isn't compatible with Loader's extended semantic version format (Could not parse version number component 'jarvis'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[14:41:54] [ForkJoinPool-1-worker-5/WARN] (FabricLoader/Metadata) "fabric.mod.json" from mod moulconfig did not have "schemaVersion" as first field.
[14:41:54] [ForkJoinPool-1-worker-4/WARN] (FabricLoader/Metadata) "fabric.mod.json" from mod rei-fake did not have "schemaVersion" as first field.
[14:41:54] [ForkJoinPool-1-worker-4/WARN] (FabricLoader/Metadata) Mod rei-fake uses the version ABC which isn't compatible with Loader's extended semantic version format (Could not parse version number component 'ABC'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[14:41:54] [ForkJoinPool-1-worker-8/WARN] (FabricLoader/Metadata) Mod org_eclipse_jgit_org_eclipse_jgit uses the version 7.0.0.202409031743-r which isn't compatible with Loader's extended semantic version format (Could not parse version number component '202409031743'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[14:41:54] [main/INFO] (FabricLoader) Loading 83 mods:
	- architectury 15.0.1
	- badoptimizations 2.2.0
	- cloth-basic-math 0.6.1
	- cloth-config 17.0.144
	- devauth 1.2.1
	- error_notifier 1.0.11
	- fabric-api 0.114.0+1.21.4
	- fabric-api-base 0.4.54+b47eab6b04
	- fabric-api-lookup-api-v1 1.6.86+b1caf1e904
	- fabric-biome-api-v1 15.0.6+b1c29d8e04
	- fabric-block-api-v1 1.0.31+7feeb73304
	- fabric-block-view-api-v2 1.0.19+7feeb73304
	- fabric-blockrenderlayer-v1 2.0.8+7feeb73304
	- fabric-client-gametest-api-v1 3.0.0+1f6471e604
	- fabric-client-tags-api-v1 1.1.29+20ea1e2304
	- fabric-command-api-v1 1.2.62+f71b366f04
	- fabric-command-api-v2 2.2.41+e496eb1504
	- fabric-commands-v0 0.2.79+df3654b304
	- fabric-content-registries-v0 9.1.14+7feeb73304
	- fabric-convention-tags-v1 2.1.11+7f945d5b04
	- fabric-convention-tags-v2 2.10.1+e7d3c6e504
	- fabric-crash-report-info-v1 0.3.6+7feeb73304
	- fabric-data-attachment-api-v1 1.4.6+9aea556b04
	- fabric-data-generation-api-v1 22.2.4+20ea1e2304
	- fabric-dimensions-v1 4.0.10+7feeb73304
	- fabric-entity-events-v1 2.0.12+60fccc7604
	- fabric-events-interaction-v0 4.0.2+a4eebcf004
	- fabric-game-rule-api-v1 1.0.62+7feeb73304
	- fabric-gametest-api-v1 2.0.21+7feeb73304
	- fabric-item-api-v1 11.1.14+203e6b2304
	- fabric-item-group-api-v1 4.1.23+7feeb73304
	- fabric-key-binding-api-v1 1.0.56+7feeb73304
	- fabric-keybindings-v0 0.2.54+df3654b304
	- fabric-language-kotlin 1.13.0+kotlin.2.1.0
	- fabric-lifecycle-events-v1 2.5.4+bf2a60eb04
	- fabric-loot-api-v2 3.0.33+3f89f5a504
	- fabric-loot-api-v3 1.0.21+203e6b2304
	- fabric-message-api-v1 6.0.25+7feeb73304
	- fabric-model-loading-api-v1 4.2.0+8ca2ae8d04
	- fabric-networking-api-v1 4.3.9+f371ccb904
	- fabric-object-builder-api-v1 18.0.8+d70d2c0604
	- fabric-particles-v1 4.0.14+7feeb73304
	- fabric-recipe-api-v1 8.0.6+9ceeb58c04
	- fabric-registry-sync-v0 6.1.3+e7d3c6e504
	- fabric-renderer-api-v1 5.0.2+84404cdd04
	- fabric-renderer-indigo 2.0.2+84404cdd04
	- fabric-rendering-data-attachment-v1 0.3.57+73761d2e04
	- fabric-rendering-fluids-v1 3.1.19+7feeb73304
	- fabric-rendering-v1 10.1.2+b874120c04
	- fabric-resource-conditions-api-v1 5.0.13+203e6b2304
	- fabric-resource-loader-v0 3.0.11+b1caf1e904
	- fabric-screen-api-v1 2.0.38+7feeb73304
	- fabric-screen-handler-api-v1 1.3.110+7feeb73304
	- fabric-sound-api-v1 1.0.32+7feeb73304
	- fabric-tag-api-v1 1.0.2+20ea1e2304
	- fabric-transfer-api-v1 5.4.9+efa825c904
	- fabric-transitive-access-wideners-v1 6.3.2+56e78b9b04
	- fabricloader 0.16.9
	- fake-jarvis-api jarvis
	- firmament 2.0.1-dev+mc1.21.4+g4562d29
	- hypixel-mod-api 1.0.1+build.1+mc1.21
	- jarviscci 1.1.4
	- java 21
	- minecraft 1.21.4
	- mixinextras 0.4.1
	- mm_shedaniel 2.4.1
	- modmenu 13.0.0-beta.1
	- moulconfig 1.0.0
	- nochatreports 1.21.4-v2.11.0
	- rei-fake ABC
	- roughlyenoughitems 18.0.796
	- skyblocker 1.23.0-beta.3+1.21.4
	   |-- com_logisticscraft_occlusionculling 0.0.8-SNAPSHOT
	   |-- hm-api 1.0.1+1.21.2
	   |-- meteordevelopment_discord-ipc 1.1
	   |-- moe_nea_neurepoparser 1.6.0
	   |-- net_azureaaron_legacy-item-dfu 1.0.2+1.21.3
	   |-- net_azureaaron_networth-calculator 1.0.5
	   |-- org_apache_commons_commons-math3 3.6.1
	   |-- org_apache_commons_commons-text 1.12.0
	   \-- org_eclipse_jgit_org_eclipse_jgit 7.0.0.202409031743-r
	- wildfire_gender 4.3.3+1.21.4
	- yet_another_config_lib_v3 3.6.2+1.21.4-fabric
[14:42:00] [main/INFO] (FabricLoader/Mixin) SpongePowered MIXIN Subsystem Version=0.8.7 Source=file:/home/nea/.gradle/caches/modules-2/files-2.1/net.fabricmc/sponge-mixin/0.15.4+mixin.0.8.7/6a12aacc794f1078458433116e9ed42c1cc98096/sponge-mixin-0.15.4+mixin.0.8.7.jar Service=Knot/Fabric Env=CLIENT
/*************************************************************************************************************************************************************************************************************************************/
/*                                                                                                        SpongePowered MIXIN                                                                                                        */
/*************************************************************************************************************************************************************************************************************************************/
/*                                         Code source : file:/home/nea/.gradle/caches/modules-2/files-2.1/net.fabricmc/sponge-mixin/0.15.4+mixin.0.8.7/6a12aacc794f1078458433116e9ed42c1cc98096/sponge-mixin-0.15.4+mixin.0.8.7.jar */
/*                                    Internal Version : 0.8.7                                                                                                                                                                       */
/*                                        Java Version : 21.0 (supports compatibility 6-21)                                                                                                                                          */
/*                         Default Compatibility Level : JAVA_8                                                                                                                                                                      */
/*                   Max Effective Compatibility Level : JAVA_13                                                                                                                                                                     */
/*                                Detected ASM Version : ASM 9.7.1 (ASM10_EXPERIMENTAL)                                                                                                                                              */
/*                          Detected ASM Supports Java : Up to Java 24 (class file version 68.0)                                                                                                                                     */
/*************************************************************************************************************************************************************************************************************************************/
/*                                        Service Name : Knot/Fabric                                                                                                                                                                 */
/*                                 Mixin Service Class : net.fabricmc.loader.impl.launch.knot.MixinServiceKnot                                                                                                                       */
/*                       Global Property Service Class : net.fabricmc.loader.impl.launch.knot.FabricGlobalPropertyService                                                                                                            */
/*                                 Logger Adapter Type : Fabric Mixin Logger                                                                                                                                                         */
/*************************************************************************************************************************************************************************************************************************************/
/*                                         mixin.debug : <false>                                                                                                                                                                     */
/*                                  mixin.debug.export : - <true>                                                                                                                                                                    */
/*                           mixin.debug.export.filter : - - <null>                                                                                                                                                                  */
/*                        mixin.debug.export.decompile : - - <true>                                                                                                                                                                  */
/*                  mixin.debug.export.decompile.async : - - - <true>                                                                                                                                                                */
/* mixin.debug.export.decompile.mergeGenericSignatures : - - - <true>                                                                                                                                                                */
/*                                  mixin.debug.verify : - <false>                                                                                                                                                                   */
/*                                 mixin.debug.verbose : - <false>                                                                                                                                                                   */
/*                         mixin.debug.countInjections : - <false>                                                                                                                                                                   */
/*                                  mixin.debug.strict : - <false>                                                                                                                                                                   */
/*                           mixin.debug.strict.unique : - - <false>                                                                                                                                                                 */
/*                          mixin.debug.strict.targets : - - <false>                                                                                                                                                                 */
/*                                mixin.debug.profiler : - <false>                                                                                                                                                                   */
/*                           mixin.dumpTargetOnFailure : <false>                                                                                                                                                                     */
/*                                        mixin.checks : <false>                                                                                                                                                                     */
/*                             mixin.checks.interfaces : - <false>                                                                                                                                                                   */
/*                      mixin.checks.interfaces.strict : - - <false>                                                                                                                                                                 */
/*                             mixin.ignoreConstraints : <false>                                                                                                                                                                     */
/*                                       mixin.hotSwap : <false>                                                                                                                                                                     */
/*                                       mixin.env.obf : - <false>                                                                                                                                                                   */
/*                             mixin.env.disableRefMap : - <false>                                                                                                                                                                   */
/*                               mixin.env.remapRefMap : - <false>                                                                                                                                                                   */
/*                       mixin.env.refMapRemappingFile : - <>                                                                                                                                                                        */
/*                        mixin.env.refMapRemappingEnv : - <searge>                                                                                                                                                                  */
/*                      mixin.env.allowPermissiveMatch : - <false>                                                                                                                                                                   */
/*                            mixin.env.ignoreRequired : - <false>                                                                                                                                                                   */
/*                               mixin.env.compatLevel : - <false>                                                                                                                                                                   */
/*                          mixin.env.shiftByViolation : - <warn>                                                                                                                                                                    */
/*                      mixin.initialiserInjectionMode : <default>                                                                                                                                                                   */
/*               mixin.tunable.classReaderExpandFrames : - <false>                                                                                                                                                                   */
/*************************************************************************************************************************************************************************************************************************************/
/*                                    UNSAFE_INJECTION : available=<true> enabled=<true>                                                                                                                                             */
/*                       INJECTORS_IN_INTERFACE_MIXINS : available=<true> enabled=<true>                                                                                                                                             */
/*************************************************************************************************************************************************************************************************************************************/
/*                                       Detected Side : CLIENT                                                                                                                                                                      */
/*************************************************************************************************************************************************************************************************************************************/
[14:42:00] [main/INFO] (FabricLoader/Mixin) Attempting to load Fernflower decompiler (Threaded mode)
[14:42:00] [main/INFO] (FabricLoader/Mixin) Fernflower could not be loaded, exported classes will not be decompiled. NoClassDefFoundError: org/jetbrains/java/decompiler/main/extern/IResultSaver
[14:42:00] [main/INFO] (FabricLoader/Mixin) Loaded Fabric development mappings for mixin remapper!
[14:42:00] [main/INFO] (FabricLoader/Mixin) Compatibility level set to JAVA_21
[14:42:00] [main/INFO] (MM-shedaniel) Started MM (shedaniel's fork) at me/shedaniel/gen/mixin/
[14:42:00] [main/WARN] (FabricLoader/Mixin) Reference map 'Manningham Mills-refmap.json' for mixins.mm_shedaniel.json could not be read. If this is a development environment you can ignore this message
[14:42:00] [main/WARN] (FabricLoader/Mixin) Reference map 'Firmament-refmap.json' for firmament.mixins.json could not be read. If this is a development environment you can ignore this message
[14:42:00] [main/INFO] (BadOptimizations) Loading config file
[14:42:00] [main/INFO] (BadOptimizations) Config version: 4
[14:42:00] [main/INFO] (BadOptimizations) BadOptimizations config dump:
[14:42:00] [main/INFO] (BadOptimizations) enable_toast_optimizations: true
[14:42:00] [main/INFO] (BadOptimizations) ignore_mod_incompatibilities: false
[14:42:00] [main/INFO] (BadOptimizations) lightmap_time_change_needed_for_update: 80
[14:42:00] [main/INFO] (BadOptimizations) enable_lightmap_caching: true
[14:42:00] [main/INFO] (BadOptimizations) enable_particle_manager_optimization: true
[14:42:00] [main/INFO] (BadOptimizations) enable_entity_renderer_caching: true
[14:42:00] [main/INFO] (BadOptimizations) log_config: true
[14:42:00] [main/INFO] (BadOptimizations) enable_remove_redundant_fov_calculations: true
[14:42:00] [main/INFO] (BadOptimizations) config_version: 4
[14:42:00] [main/INFO] (BadOptimizations) enable_sky_angle_caching_in_worldrenderer: true
[14:42:00] [main/INFO] (BadOptimizations) enable_block_entity_renderer_caching: true
[14:42:00] [main/INFO] (BadOptimizations) skycolor_time_change_needed_for_update: 3
[14:42:00] [main/INFO] (BadOptimizations) enable_entity_flag_caching: true
[14:42:00] [main/INFO] (BadOptimizations) enable_debug_renderer_disable_if_not_needed: true
[14:42:00] [main/INFO] (BadOptimizations) enable_sky_color_caching: true
[14:42:00] [main/INFO] (BadOptimizations) enable_remove_tutorial_if_not_demo: true
[14:42:00] [main/INFO] (BadOptimizations) show_f3_text: true
[14:42:00] [main/INFO] (BadOptimizations) Note: Entity flag caching has no effect in Minecraft 1.20.5+ as the optimization is now in vanilla.
Trying to discover mixins
Found classes at file:/home/nea/src/Firmament/build/classes/java/main/
Base directory found at /home/nea/src/Firmament/build/classes/java/main
Trying to find mixins from directory
Found mixins: [CaseInsensitiveCommandMapPatch, SaveCursorPositionPatch, AppendRepoAsResourcePack, devenv.IdentifyCloser, devenv.DisableCommonPacketWarnings, devenv.DisableInvalidFishingHook, devenv.MixinScoreboard, devenv.IdentifyStopperPatch, devenv.WarnForUnknownCustomPayloadSends, devenv.EarlyInstantiateTranslations, devenv.MixinKeyboard, devenv.WarnOnMissingTranslations, accessor.AccessorAbstractClientPlayerEntity, accessor.AccessorHandledScreen, accessor.AccessorChatHud, BandAidResourcePackPatch, customgui.OriginalSlotCoords, customgui.PatchHandledScreen, TextureUnpackBase64PadPatch, DFUEntityIdFixPatch, RedirectWithoutSubCommands, LenientProfileComponentPatch, AlwaysDisplayFirmamentClientCommandErrors, FirmKeybindsInVanillaControlsPatch, TolerateFirmamentTolerateRegistryOwners, SlotClickEventPatch, EarlyResourceReloadPatch, ResourceReloaderRegistrationPatch, ChatPeekingPatch, KeyPressInWorldEventPatch, OutgoingPacketEventPatch, PlayerDropEventPatch, PropertySignatureIgnorePatch, ScreenChangeEventPatch, ToggleSprintPatch, YggdrasilSignatureIgnorePatch, SoundReceiveEventPatch, HudRenderEventsPatch, EntityInteractEventPatch, EntityDespawnPatch, PropertySignatureIgnorePatchForSession, EntityUpdateEventListener, CustomDurabilityBarPatch, MainWindowFirstLoadPatch, MixinHandledScreen, SlotUpdateListener, WorldReadyEventPatch, WorldRenderLastEventPatch, IncomingPacketListenerPatches, HideStatusEffectsPatch, SaveOriginalCommandTreePacket]
Found firmament class roots: /home/nea/src/Firmament/build/classes/java/configured:/home/nea/src/Firmament/build/classes/kotlin/configured:/home/nea/src/Firmament/build/classes/java/sodium:/home/nea/src/Firmament/build/classes/kotlin/sodium:/home/nea/src/Firmament/build/classes/java/citresewn:/home/nea/src/Firmament/build/classes/kotlin/citresewn:/home/nea/src/Firmament/build/classes/java/yacl:/home/nea/src/Firmament/build/classes/kotlin/yacl:/home/nea/src/Firmament/build/classes/java/explosiveEnhancement:/home/nea/src/Firmament/build/classes/kotlin/explosiveEnhancement:/home/nea/src/Firmament/build/classes/java/wildfireGender:/home/nea/src/Firmament/build/classes/kotlin/wildfireGender:/home/nea/src/Firmament/build/classes/java/modmenu:/home/nea/src/Firmament/build/classes/kotlin/modmenu:/home/nea/src/Firmament/build/classes/java/rei:/home/nea/src/Firmament/build/classes/kotlin/rei:/home/nea/src/Firmament/build/classes/java/moulconfig:/home/nea/src/Firmament/build/classes/kotlin/moulconfig:/home/nea/src/Firmament/build/classes/java/texturePacks:/home/nea/src/Firmament/build/classes/kotlin/texturePacks
Base directory found at /home/nea/src/Firmament/build/classes/java/configured
Skipping non-existing mixin root: /home/nea/src/Firmament/build/classes/java/configured
Base directory found at /home/nea/src/Firmament/build/classes/kotlin/configured
Skipping non-existing mixin root: /home/nea/src/Firmament/build/classes/kotlin/configured
Base directory found at /home/nea/src/Firmament/build/classes/java/sodium
Skipping non-existing mixin root: /home/nea/src/Firmament/build/classes/java/sodium
Base directory found at /home/nea/src/Firmament/build/classes/kotlin/sodium
Skipping non-existing mixin root: /home/nea/src/Firmament/build/classes/kotlin/sodium
Base directory found at /home/nea/src/Firmament/build/classes/java/citresewn
Skipping non-existing mixin root: /home/nea/src/Firmament/build/classes/java/citresewn
Base directory found at /home/nea/src/Firmament/build/classes/kotlin/citresewn
Skipping non-existing mixin root: /home/nea/src/Firmament/build/classes/kotlin/citresewn
Base directory found at /home/nea/src/Firmament/build/classes/java/yacl
Skipping non-existing mixin root: /home/nea/src/Firmament/build/classes/java/yacl
Base directory found at /home/nea/src/Firmament/build/classes/kotlin/yacl
Trying to find mixins from directory
Found mixins: [CaseInsensitiveCommandMapPatch, SaveCursorPositionPatch, AppendRepoAsResourcePack, devenv.IdentifyCloser, devenv.DisableCommonPacketWarnings, devenv.DisableInvalidFishingHook, devenv.MixinScoreboard, devenv.IdentifyStopperPatch, devenv.WarnForUnknownCustomPayloadSends, devenv.EarlyInstantiateTranslations, devenv.MixinKeyboard, devenv.WarnOnMissingTranslations, accessor.AccessorAbstractClientPlayerEntity, accessor.AccessorHandledScreen, accessor.AccessorChatHud, BandAidResourcePackPatch, customgui.OriginalSlotCoords, customgui.PatchHandledScreen, TextureUnpackBase64PadPatch, DFUEntityIdFixPatch, RedirectWithoutSubCommands, LenientProfileComponentPatch, AlwaysDisplayFirmamentClientCommandErrors, FirmKeybindsInVanillaControlsPatch, TolerateFirmamentTolerateRegistryOwners, SlotClickEventPatch, EarlyResourceReloadPatch, ResourceReloaderRegistrationPatch, ChatPeekingPatch, KeyPressInWorldEventPatch, OutgoingPacketEventPatch, PlayerDropEventPatch, PropertySignatureIgnorePatch, ScreenChangeEventPatch, ToggleSprintPatch, YggdrasilSignatureIgnorePatch, SoundReceiveEventPatch, HudRenderEventsPatch, EntityInteractEventPatch, EntityDespawnPatch, PropertySignatureIgnorePatchForSession, EntityUpdateEventListener, CustomDurabilityBarPatch, MainWindowFirstLoadPatch, MixinHandledScreen, SlotUpdateListener, WorldReadyEventPatch, WorldRenderLastEventPatch, IncomingPacketListenerPatches, HideStatusEffectsPatch, SaveOriginalCommandTreePacket]
Base directory found at /home/nea/src/Firmament/build/classes/java/explosiveEnhancement
Skipping non-existing mixin root: /home/nea/src/Firmament/build/classes/java/explosiveEnhancement
Base directory found at /home/nea/src/Firmament/build/classes/kotlin/explosiveEnhancement
Skipping non-existing mixin root: /home/nea/src/Firmament/build/classes/kotlin/explosiveEnhancement
Base directory found at /home/nea/src/Firmament/build/classes/java/wildfireGender
Trying to find mixins from directory
Found mixins: [CaseInsensitiveCommandMapPatch, SaveCursorPositionPatch, AppendRepoAsResourcePack, devenv.IdentifyCloser, devenv.DisableCommonPacketWarnings, devenv.DisableInvalidFishingHook, devenv.MixinScoreboard, devenv.IdentifyStopperPatch, devenv.WarnForUnknownCustomPayloadSends, devenv.EarlyInstantiateTranslations, devenv.MixinKeyboard, devenv.WarnOnMissingTranslations, accessor.AccessorAbstractClientPlayerEntity, accessor.AccessorHandledScreen, accessor.AccessorChatHud, BandAidResourcePackPatch, customgui.OriginalSlotCoords, customgui.PatchHandledScreen, TextureUnpackBase64PadPatch, DFUEntityIdFixPatch, RedirectWithoutSubCommands, LenientProfileComponentPatch, AlwaysDisplayFirmamentClientCommandErrors, FirmKeybindsInVanillaControlsPatch, TolerateFirmamentTolerateRegistryOwners, SlotClickEventPatch, EarlyResourceReloadPatch, ResourceReloaderRegistrationPatch, ChatPeekingPatch, KeyPressInWorldEventPatch, OutgoingPacketEventPatch, PlayerDropEventPatch, PropertySignatureIgnorePatch, ScreenChangeEventPatch, ToggleSprintPatch, YggdrasilSignatureIgnorePatch, SoundReceiveEventPatch, HudRenderEventsPatch, EntityInteractEventPatch, EntityDespawnPatch, PropertySignatureIgnorePatchForSession, EntityUpdateEventListener, CustomDurabilityBarPatch, MainWindowFirstLoadPatch, MixinHandledScreen, SlotUpdateListener, WorldReadyEventPatch, WorldRenderLastEventPatch, IncomingPacketListenerPatches, HideStatusEffectsPatch, SaveOriginalCommandTreePacket, compat.wildfiregender.PatchArmorTexturesInGenderMod]
Base directory found at /home/nea/src/Firmament/build/classes/kotlin/wildfireGender
Skipping non-existing mixin root: /home/nea/src/Firmament/build/classes/kotlin/wildfireGender
Base directory found at /home/nea/src/Firmament/build/classes/java/modmenu
Skipping non-existing mixin root: /home/nea/src/Firmament/build/classes/java/modmenu
Base directory found at /home/nea/src/Firmament/build/classes/kotlin/modmenu
Trying to find mixins from directory
Found mixins: [CaseInsensitiveCommandMapPatch, SaveCursorPositionPatch, AppendRepoAsResourcePack, devenv.IdentifyCloser, devenv.DisableCommonPacketWarnings, devenv.DisableInvalidFishingHook, devenv.MixinScoreboard, devenv.IdentifyStopperPatch, devenv.WarnForUnknownCustomPayloadSends, devenv.EarlyInstantiateTranslations, devenv.MixinKeyboard, devenv.WarnOnMissingTranslations, accessor.AccessorAbstractClientPlayerEntity, accessor.AccessorHandledScreen, accessor.AccessorChatHud, BandAidResourcePackPatch, customgui.OriginalSlotCoords, customgui.PatchHandledScreen, TextureUnpackBase64PadPatch, DFUEntityIdFixPatch, RedirectWithoutSubCommands, LenientProfileComponentPatch, AlwaysDisplayFirmamentClientCommandErrors, FirmKeybindsInVanillaControlsPatch, TolerateFirmamentTolerateRegistryOwners, SlotClickEventPatch, EarlyResourceReloadPatch, ResourceReloaderRegistrationPatch, ChatPeekingPatch, KeyPressInWorldEventPatch, OutgoingPacketEventPatch, PlayerDropEventPatch, PropertySignatureIgnorePatch, ScreenChangeEventPatch, ToggleSprintPatch, YggdrasilSignatureIgnorePatch, SoundReceiveEventPatch, HudRenderEventsPatch, EntityInteractEventPatch, EntityDespawnPatch, PropertySignatureIgnorePatchForSession, EntityUpdateEventListener, CustomDurabilityBarPatch, MainWindowFirstLoadPatch, MixinHandledScreen, SlotUpdateListener, WorldReadyEventPatch, WorldRenderLastEventPatch, IncomingPacketListenerPatches, HideStatusEffectsPatch, SaveOriginalCommandTreePacket, compat.wildfiregender.PatchArmorTexturesInGenderMod]
Base directory found at /home/nea/src/Firmament/build/classes/java/rei
Trying to find mixins from directory
Found mixins: [CaseInsensitiveCommandMapPatch, SaveCursorPositionPatch, AppendRepoAsResourcePack, devenv.IdentifyCloser, devenv.DisableCommonPacketWarnings, devenv.DisableInvalidFishingHook, devenv.MixinScoreboard, devenv.IdentifyStopperPatch, devenv.WarnForUnknownCustomPayloadSends, devenv.EarlyInstantiateTranslations, devenv.MixinKeyboard, devenv.WarnOnMissingTranslations, accessor.AccessorAbstractClientPlayerEntity, accessor.AccessorHandledScreen, accessor.AccessorChatHud, BandAidResourcePackPatch, customgui.OriginalSlotCoords, customgui.PatchHandledScreen, TextureUnpackBase64PadPatch, DFUEntityIdFixPatch, RedirectWithoutSubCommands, LenientProfileComponentPatch, AlwaysDisplayFirmamentClientCommandErrors, FirmKeybindsInVanillaControlsPatch, TolerateFirmamentTolerateRegistryOwners, SlotClickEventPatch, EarlyResourceReloadPatch, ResourceReloaderRegistrationPatch, ChatPeekingPatch, KeyPressInWorldEventPatch, OutgoingPacketEventPatch, PlayerDropEventPatch, PropertySignatureIgnorePatch, ScreenChangeEventPatch, ToggleSprintPatch, YggdrasilSignatureIgnorePatch, SoundReceiveEventPatch, HudRenderEventsPatch, EntityInteractEventPatch, EntityDespawnPatch, PropertySignatureIgnorePatchForSession, EntityUpdateEventListener, CustomDurabilityBarPatch, MainWindowFirstLoadPatch, MixinHandledScreen, SlotUpdateListener, WorldReadyEventPatch, WorldRenderLastEventPatch, IncomingPacketListenerPatches, HideStatusEffectsPatch, SaveOriginalCommandTreePacket, compat.wildfiregender.PatchArmorTexturesInGenderMod, compat.HideREIRecipeWarning]
Base directory found at /home/nea/src/Firmament/build/classes/kotlin/rei
Trying to find mixins from directory
Found mixins: [CaseInsensitiveCommandMapPatch, SaveCursorPositionPatch, AppendRepoAsResourcePack, devenv.IdentifyCloser, devenv.DisableCommonPacketWarnings, devenv.DisableInvalidFishingHook, devenv.MixinScoreboard, devenv.IdentifyStopperPatch, devenv.WarnForUnknownCustomPayloadSends, devenv.EarlyInstantiateTranslations, devenv.MixinKeyboard, devenv.WarnOnMissingTranslations, accessor.AccessorAbstractClientPlayerEntity, accessor.AccessorHandledScreen, accessor.AccessorChatHud, BandAidResourcePackPatch, customgui.OriginalSlotCoords, customgui.PatchHandledScreen, TextureUnpackBase64PadPatch, DFUEntityIdFixPatch, RedirectWithoutSubCommands, LenientProfileComponentPatch, AlwaysDisplayFirmamentClientCommandErrors, FirmKeybindsInVanillaControlsPatch, TolerateFirmamentTolerateRegistryOwners, SlotClickEventPatch, EarlyResourceReloadPatch, ResourceReloaderRegistrationPatch, ChatPeekingPatch, KeyPressInWorldEventPatch, OutgoingPacketEventPatch, PlayerDropEventPatch, PropertySignatureIgnorePatch, ScreenChangeEventPatch, ToggleSprintPatch, YggdrasilSignatureIgnorePatch, SoundReceiveEventPatch, HudRenderEventsPatch, EntityInteractEventPatch, EntityDespawnPatch, PropertySignatureIgnorePatchForSession, EntityUpdateEventListener, CustomDurabilityBarPatch, MainWindowFirstLoadPatch, MixinHandledScreen, SlotUpdateListener, WorldReadyEventPatch, WorldRenderLastEventPatch, IncomingPacketListenerPatches, HideStatusEffectsPatch, SaveOriginalCommandTreePacket, compat.wildfiregender.PatchArmorTexturesInGenderMod, compat.HideREIRecipeWarning]
Base directory found at /home/nea/src/Firmament/build/classes/java/moulconfig
Skipping non-existing mixin root: /home/nea/src/Firmament/build/classes/java/moulconfig
Base directory found at /home/nea/src/Firmament/build/classes/kotlin/moulconfig
Trying to find mixins from directory
Found mixins: [CaseInsensitiveCommandMapPatch, SaveCursorPositionPatch, AppendRepoAsResourcePack, devenv.IdentifyCloser, devenv.DisableCommonPacketWarnings, devenv.DisableInvalidFishingHook, devenv.MixinScoreboard, devenv.IdentifyStopperPatch, devenv.WarnForUnknownCustomPayloadSends, devenv.EarlyInstantiateTranslations, devenv.MixinKeyboard, devenv.WarnOnMissingTranslations, accessor.AccessorAbstractClientPlayerEntity, accessor.AccessorHandledScreen, accessor.AccessorChatHud, BandAidResourcePackPatch, customgui.OriginalSlotCoords, customgui.PatchHandledScreen, TextureUnpackBase64PadPatch, DFUEntityIdFixPatch, RedirectWithoutSubCommands, LenientProfileComponentPatch, AlwaysDisplayFirmamentClientCommandErrors, FirmKeybindsInVanillaControlsPatch, TolerateFirmamentTolerateRegistryOwners, SlotClickEventPatch, EarlyResourceReloadPatch, ResourceReloaderRegistrationPatch, ChatPeekingPatch, KeyPressInWorldEventPatch, OutgoingPacketEventPatch, PlayerDropEventPatch, PropertySignatureIgnorePatch, ScreenChangeEventPatch, ToggleSprintPatch, YggdrasilSignatureIgnorePatch, SoundReceiveEventPatch, HudRenderEventsPatch, EntityInteractEventPatch, EntityDespawnPatch, PropertySignatureIgnorePatchForSession, EntityUpdateEventListener, CustomDurabilityBarPatch, MainWindowFirstLoadPatch, MixinHandledScreen, SlotUpdateListener, WorldReadyEventPatch, WorldRenderLastEventPatch, IncomingPacketListenerPatches, HideStatusEffectsPatch, SaveOriginalCommandTreePacket, compat.wildfiregender.PatchArmorTexturesInGenderMod, compat.HideREIRecipeWarning]
Base directory found at /home/nea/src/Firmament/build/classes/java/texturePacks
Trying to find mixins from directory
Found mixins: [CaseInsensitiveCommandMapPatch, SaveCursorPositionPatch, AppendRepoAsResourcePack, devenv.IdentifyCloser, devenv.DisableCommonPacketWarnings, devenv.DisableInvalidFishingHook, devenv.MixinScoreboard, devenv.IdentifyStopperPatch, devenv.WarnForUnknownCustomPayloadSends, devenv.EarlyInstantiateTranslations, devenv.MixinKeyboard, devenv.WarnOnMissingTranslations, accessor.AccessorAbstractClientPlayerEntity, accessor.AccessorHandledScreen, accessor.AccessorChatHud, BandAidResourcePackPatch, customgui.OriginalSlotCoords, customgui.PatchHandledScreen, TextureUnpackBase64PadPatch, DFUEntityIdFixPatch, RedirectWithoutSubCommands, LenientProfileComponentPatch, AlwaysDisplayFirmamentClientCommandErrors, FirmKeybindsInVanillaControlsPatch, TolerateFirmamentTolerateRegistryOwners, SlotClickEventPatch, EarlyResourceReloadPatch, ResourceReloaderRegistrationPatch, ChatPeekingPatch, KeyPressInWorldEventPatch, OutgoingPacketEventPatch, PlayerDropEventPatch, PropertySignatureIgnorePatch, ScreenChangeEventPatch, ToggleSprintPatch, YggdrasilSignatureIgnorePatch, SoundReceiveEventPatch, HudRenderEventsPatch, EntityInteractEventPatch, EntityDespawnPatch, PropertySignatureIgnorePatchForSession, EntityUpdateEventListener, CustomDurabilityBarPatch, MainWindowFirstLoadPatch, MixinHandledScreen, SlotUpdateListener, WorldReadyEventPatch, WorldRenderLastEventPatch, IncomingPacketListenerPatches, HideStatusEffectsPatch, SaveOriginalCommandTreePacket, compat.wildfiregender.PatchArmorTexturesInGenderMod, compat.HideREIRecipeWarning, custommodels.ApplyHeadModelInItemRenderer, custommodels.ReplaceBlockBreakSoundPatch, custommodels.SupplyFakeModelPatch, custommodels.PatchArmorTexture, custommodels.PatchLegacyArmorLayerSupport, custommodels.ReplaceBlockHitSoundPatch, custommodels.ReplaceBlockRenderManagerBlockModel, custommodels.ReplaceFallbackBlockModel, custommodels.ReplaceTextColorInHandledScreen, custommodels.CustomSkullTexturePatch, custommodels.ReplaceItemModelPatch]
Base directory found at /home/nea/src/Firmament/build/classes/kotlin/texturePacks
Trying to find mixins from directory
Found mixins: [CaseInsensitiveCommandMapPatch, SaveCursorPositionPatch, AppendRepoAsResourcePack, devenv.IdentifyCloser, devenv.DisableCommonPacketWarnings, devenv.DisableInvalidFishingHook, devenv.MixinScoreboard, devenv.IdentifyStopperPatch, devenv.WarnForUnknownCustomPayloadSends, devenv.EarlyInstantiateTranslations, devenv.MixinKeyboard, devenv.WarnOnMissingTranslations, accessor.AccessorAbstractClientPlayerEntity, accessor.AccessorHandledScreen, accessor.AccessorChatHud, BandAidResourcePackPatch, customgui.OriginalSlotCoords, customgui.PatchHandledScreen, TextureUnpackBase64PadPatch, DFUEntityIdFixPatch, RedirectWithoutSubCommands, LenientProfileComponentPatch, AlwaysDisplayFirmamentClientCommandErrors, FirmKeybindsInVanillaControlsPatch, TolerateFirmamentTolerateRegistryOwners, SlotClickEventPatch, EarlyResourceReloadPatch, ResourceReloaderRegistrationPatch, ChatPeekingPatch, KeyPressInWorldEventPatch, OutgoingPacketEventPatch, PlayerDropEventPatch, PropertySignatureIgnorePatch, ScreenChangeEventPatch, ToggleSprintPatch, YggdrasilSignatureIgnorePatch, SoundReceiveEventPatch, HudRenderEventsPatch, EntityInteractEventPatch, EntityDespawnPatch, PropertySignatureIgnorePatchForSession, EntityUpdateEventListener, CustomDurabilityBarPatch, MainWindowFirstLoadPatch, MixinHandledScreen, SlotUpdateListener, WorldReadyEventPatch, WorldRenderLastEventPatch, IncomingPacketListenerPatches, HideStatusEffectsPatch, SaveOriginalCommandTreePacket, compat.wildfiregender.PatchArmorTexturesInGenderMod, compat.HideREIRecipeWarning, custommodels.ApplyHeadModelInItemRenderer, custommodels.ReplaceBlockBreakSoundPatch, custommodels.SupplyFakeModelPatch, custommodels.PatchArmorTexture, custommodels.PatchLegacyArmorLayerSupport, custommodels.ReplaceBlockHitSoundPatch, custommodels.ReplaceBlockRenderManagerBlockModel, custommodels.ReplaceFallbackBlockModel, custommodels.ReplaceTextColorInHandledScreen, custommodels.CustomSkullTexturePatch, custommodels.ReplaceItemModelPatch]
[14:42:01] [main/INFO] (FabricLoader/MixinExtras|Service) Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.4.1).
[14:42:02] [main/INFO] (DevAuth) Successfully logged in as lrg89
[14:42:04] [Datafixer Bootstrap/INFO] (Minecraft) 243 Datafixer optimizations took 317 milliseconds
[14:42:09] [Render thread/INFO] (Minecraft) Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[14:42:09] [Render thread/INFO] (Minecraft) Setting user: lrg89
[14:42:09] [Render thread/INFO] (MM-shedaniel) Definitely not up to no good
[14:42:09] [Render thread/INFO] (REI) [REI] Config loaded
[14:42:09] [Render thread/INFO] (REI) [REI] Registered plugin provider FirmamentReiCommonPlugin [firmament] for REICommonPlugin
[14:42:09] [Render thread/INFO] (NetworkManagerImpl) Registering S2C receiver with id roughlyenoughitems:request_tags_s2c
[14:42:09] [Render thread/INFO] (NetworkManagerImpl) Registering C2S receiver with id roughlyenoughitems:request_tags_c2s
[14:42:09] [Render thread/INFO] (REI) [REI] Registered plugin provider DefaultPlugin [roughlyenoughitems] for REICommonPlugin
[14:42:09] [Render thread/INFO] (REI) [REI] Registered plugin provider DefaultRuntimePlugin [roughlyenoughitems] for REICommonPlugin
[14:42:09] [Render thread/INFO] (REI) [REI] Registered plugin provider FabricFluidAPISupportPlugin for REICommonPlugin
[14:42:09] [Render thread/INFO] (NetworkManagerImpl) Registering C2S receiver with id roughlyenoughitems:delete_item
[14:42:09] [Render thread/INFO] (NetworkManagerImpl) Registering C2S receiver with id roughlyenoughitems:create_item
[14:42:09] [Render thread/INFO] (NetworkManagerImpl) Registering C2S receiver with id roughlyenoughitems:create_item_grab
[14:42:09] [Render thread/INFO] (NetworkManagerImpl) Registering C2S receiver with id roughlyenoughitems:create_item_hotbar
[14:42:09] [Render thread/INFO] (NetworkManagerImpl) Registering C2S receiver with id roughlyenoughitems:move_items_new
[14:42:09] [Render thread/INFO] (NetworkManagerImpl) Registering S2C receiver with id architectury:spawn_entity_packet
[14:42:09] [Render thread/INFO] (YetAnotherConfigLib) Deserializing YACLConfig from '/home/nea/src/Firmament/run/config/yacl.json5'
[14:42:10] [Render thread/INFO] (Mod Menu/Update Checker) Checking mod updates...
[14:42:10] [ModMenu/Update Checker/Fabric Loader/INFO] (Mod Menu/Update Checker) Update available for 'fabricloader@0.16.9'
[14:42:10] [Worker-Main-1/INFO] (Mod Menu/Update Checker) Update available for 'badoptimizations@2.2.0', (-> 2.2.1)
[14:42:11] [DefaultDispatcher-worker-1 @Firmament#2/INFO] (HypixelStaticData) Updating collection data
[14:42:11] [DefaultDispatcher-worker-3 @Firmament#3/INFO] (HypixelStaticData) Updating NEU prices
[14:42:11] [Render thread/INFO] (FabricModAPI) Registered clientbound packet with identifier 'hypixel:hello'
[14:42:11] [Render thread/INFO] (FabricModAPI) Registered clientbound packet with identifier 'hypixel:player_info'
[14:42:11] [Render thread/INFO] (FabricModAPI) Registered clientbound packet with identifier 'hyevent:location'
[14:42:11] [Render thread/INFO] (FabricModAPI) Registered clientbound packet with identifier 'hypixel:party_info'
[14:42:11] [Render thread/INFO] (FabricModAPI) Registered clientbound packet with identifier 'hypixel:ping'
[14:42:11] [Render thread/INFO] (FabricModAPI) Registered serverbound packet with identifier 'hypixel:player_info'
[14:42:11] [Render thread/INFO] (FabricModAPI) Registered serverbound packet with identifier 'hypixel:party_info'
[14:42:11] [Render thread/INFO] (FabricModAPI) Registered serverbound packet with identifier 'hypixel:ping'
[14:42:11] [Render thread/INFO] (FabricModAPI) Registered serverbound packet with identifier 'hypixel:register'
[14:42:11] [Render thread/INFO] (FabricModAPI) Debug mode is enabled!
[14:42:11] [Render thread/WARN] (FabricModAPI) Failed to send a packet as the client is not connected to a server 'ServerboundRegisterPacket{wantedPackets={hyevent:location=1}} VersionedPacket{version=1}'
[14:42:11] [Render thread/INFO] (YetAnotherConfigLib) Deserializing SkyblockerConfig from '/home/nea/src/Firmament/run/config/skyblocker.json'
[14:42:11] [Render thread/INFO] (YetAnotherConfigLib) Serializing class de.hysky.skyblocker.config.SkyblockerConfig to '/home/nea/src/Firmament/run/config/skyblocker.json'
[14:42:11] [ForkJoinPool.commonPool-worker-3/WARN] (ChatRule) [Skyblocker Chat Rules] chat rules file not found, using default rules. This is normal when using for the first time.
[14:42:11] [ForkJoinPool.commonPool-worker-1/ERROR] (NEURepoManager) [Skyblocker NEU Repo] Transport operation failed. Most likely unable to connect to the remote NEU repo on github
 org.eclipse.jgit.api.errors.TransportException: git@github.com:NotEnoughUpdates/NotEnoughUpdates-REPO.git: remote hung up unexpectedly
	at knot/org.eclipse.jgit.api.FetchCommand.call(FetchCommand.java:249) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
	at knot/org.eclipse.jgit.api.CloneCommand.fetch(CloneCommand.java:319) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
	at knot/org.eclipse.jgit.api.CloneCommand.call(CloneCommand.java:189) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
	at knot/de.hysky.skyblocker.utils.NEURepoManager.lambda$loadRepository$3(NEURepoManager.java:92) ~[skyblocker-1.23.0-beta.3+1.21.4-64b5720b4b825f21.jar:?]
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run$$$capture(CompletableFuture.java:1768) [?:?]
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java) [?:?]
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) [?:?]
	at java.base/java.util.concurrent.ForkJoinTask.doExec$$$capture(ForkJoinTask.java:507) [?:?]
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java) [?:?]
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1491) [?:?]
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:2073) [?:?]
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2035) [?:?]
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:187) [?:?]
Caused by: org.eclipse.jgit.errors.TransportException: git@github.com:NotEnoughUpdates/NotEnoughUpdates-REPO.git: remote hung up unexpectedly
	at knot/org.eclipse.jgit.transport.TransportGitSsh$SshFetchConnection.<init>(TransportGitSsh.java:317) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
	at knot/org.eclipse.jgit.transport.TransportGitSsh.openFetch(TransportGitSsh.java:152) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
	at knot/org.eclipse.jgit.transport.FetchProcess.executeImp(FetchProcess.java:153) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
Caused by: org.eclipse.jgit.errors.TransportException: git@github.com:NotEnoughUpdates/NotEnoughUpdates-REPO.git: remote hung up unexpectedly

	at knot/org.eclipse.jgit.transport.FetchProcess.execute(FetchProcess.java:105) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
	at knot/org.eclipse.jgit.transport.Transport.fetch(Transport.java:1458) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
	at knot/org.eclipse.jgit.api.FetchCommand.call(FetchCommand.java:238) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
	... 12 more
Caused by: java.lang.NullPointerException: Cannot invoke "org.eclipse.jgit.transport.SshSessionFactory.getSession(org.eclipse.jgit.transport.URIish, org.eclipse.jgit.transport.CredentialsProvider, org.eclipse.jgit.util.FS, int)" because "this.sch" is null
	at knot/org.eclipse.jgit.transport.SshTransport.getSession(SshTransport.java:107) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
	at knot/org.eclipse.jgit.transport.TransportGitSsh$SshFetchConnection.<init>(TransportGitSsh.java:285) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
	at knot/org.eclipse.jgit.transport.TransportGitSsh.openFetch(TransportGitSsh.java:152) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
	at knot/org.eclipse.jgit.transport.FetchProcess.executeImp(FetchProcess.java:153) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
Caused by: java.lang.NullPointerException: Cannot invoke "org.eclipse.jgit.transport.SshSessionFactory.getSession(org.eclipse.jgit.transport.URIish, org.eclipse.jgit.transport.CredentialsProvider, org.eclipse.jgit.util.FS, int)" because "this.sch" is null

	at knot/org.eclipse.jgit.transport.FetchProcess.execute(FetchProcess.java:105) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
	at knot/org.eclipse.jgit.transport.Transport.fetch(Transport.java:1458) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
	at knot/org.eclipse.jgit.api.FetchCommand.call(FetchCommand.java:238) ~[org_eclipse_jgit_org_eclipse_jgi409031743-r-4162cd2178a4fdba.jar:?]
	... 12 more
[14:42:11] [ForkJoinPool.commonPool-worker-1/ERROR] (NEURepoManager) [Skyblocker NEU Repo] Encountered unknown exception while loading NEU Repository
 io.github.moulberry.repo.NEURepositoryException: NEU Repository | Failure loading constants/bonuses.json: this file is required
	at knot/io.github.moulberry.repo.NEURepository.requireFile(NEURepository.java:159) ~[moe_nea_neurepoparser-1.6.0-2e60da821c94551f.jar:?]
	at knot/io.github.moulberry.repo.NEUConstants.reload(NEUConstants.java:28) ~[moe_nea_neurepoparser-1.6.0-2e60da821c94551f.jar:?]
	at knot/io.github.moulberry.repo.NEURepository.reload(NEURepository.java:131) ~[moe_nea_neurepoparser-1.6.0-2e60da821c94551f.jar:?]
	at knot/de.hysky.skyblocker.utils.NEURepoManager.lambda$loadRepository$3(NEURepoManager.java:108) ~[skyblocker-1.23.0-beta.3+1.21.4-64b5720b4b825f21.jar:?]
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run$$$capture(CompletableFuture.java:1768) [?:?]
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java) [?:?]
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) [?:?]
	at java.base/java.util.concurrent.ForkJoinTask.doExec$$$capture(ForkJoinTask.java:507) [?:?]
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java) [?:?]
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1491) [?:?]
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:2073) [?:?]
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2035) [?:?]
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:187) [?:?]
[14:42:11] [ForkJoinPool.commonPool-worker-5/ERROR] (SackItemAutocomplete) [Skyblocker Sack Item Autocomplete] Failed to load sacks data from the NEU Repo.
 java.lang.NullPointerException: Cannot invoke "io.github.moulberry.repo.NEURepoFile.stream()" because the return value of "io.github.moulberry.repo.NEURepository.file(String)" is null
	at knot/de.hysky.skyblocker.skyblock.SackItemAutocomplete.loadSackItems(SackItemAutocomplete.java:47) ~[skyblocker-1.23.0-beta.3+1.21.4-64b5720b4b825f21.jar:?]
	at java.base/java.util.concurrent.CompletableFuture$UniRun.tryFire$$$capture(CompletableFuture.java:787) [?:?]
	at java.base/java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java) [?:?]
	at java.base/java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:483) [?:?]
	at java.base/java.util.concurrent.ForkJoinTask.doExec$$$capture(ForkJoinTask.java:507) [?:?]
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java) [?:?]
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1491) [?:?]
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:2073) [?:?]
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2035) [?:?]
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:187) [?:?]
	```

</details>

### Minecraft Version

1.21.4

### Skyblocker Version

1.23.0-beta-3+mc1.21.4

### Additional context

_No response_

--- a/src/main/java/de/hysky/skyblocker/mixins/jgit/UrlConfigMixin.java
+++ b/src/main/java/de/hysky/skyblocker/mixins/jgit/UrlConfigMixin.java
@@ -1,0 +1,19 @@
+package de.hysky.skyblocker.mixins.jgit;
+
+import java.util.Map;
+
+import org.eclipse.jgit.transport.UrlConfig;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+
+import com.llamalad7.mixinextras.injector.wrapoperation.Operation;
+import com.llamalad7.mixinextras.injector.wrapoperation.WrapOperation;
+
+@Mixin(value = UrlConfig.class, remap = false)
+public class UrlConfigMixin {
+
+	@WrapOperation(method = "load", at = @At(value = "INVOKE", target = "Ljava/util/Map;put(Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object;"))
+	private Object skyblocker$ignoreUrlRedirects(Map<String, String> map, Object key, Object value, Operation<Object> operation) {
+		return null;
+	}
+}

--- a/src/main/resources/skyblocker.mixins.json
+++ b/src/main/resources/skyblocker.mixins.json
@@ -62,7 +62,8 @@
     "accessors.SlotAccessor",
     "accessors.WorldRendererAccessor",
     "discordipc.ConnectionMixin",
-    "jgit.SystemReaderMixin"
+    "jgit.SystemReaderMixin",
+    "jgit.UrlConfigMixin"
   ],
   "injectors": {
     "defaultRequire": 1

